sections/main-cart-items.liquid

```diff
@@ -302,6 +304,34 @@
       "default": false,
       "label": "t:sections.main-cart-items.settings.show_vendor.label"
     },
+    {
+      "type": "select",
+      "id": "color_scheme",
+      "options": [
+        {
+          "value": "accent-1",
+          "label": "t:sections.all.colors.accent_1.label"
+        },
+        {
+          "value": "accent-2",
+          "label": "t:sections.all.colors.accent_2.label"
+        },
+        {
+          "value": "background-1",
+          "label": "t:sections.all.colors.background_1.label"
+        },
+        {
+          "value": "background-2",
+          "label": "t:sections.all.colors.background_2.label"
+        },
+        {
+          "value": "inverse",
+          "label": "t:sections.all.colors.inverse.label"
+        }
+      ],
+      "default": "background-1",
+      "label": "t:sections.all.colors.label"
+    },
```

Comment:
Should there be an option for Subtotal as well ? Otherwise it looks a bit visually bad because subtotal looks disconnected from the main items: 
![](https://screenshot.click/22-01-w776p-gwy0f.png)